incubator/hnc/pkg/controllers/object_controller_test.go

```diff
@@ -34,6 +38,9 @@ var _ = Describe("Secret", func() {
 	})
 
 	It("should be copied to descendents", func() {
+		// Add secret to HNCConfiguration so that an ObjectReconciler will be created for secret.
+		addSecretToHNCConfig(ctx, hncConfig)
+
```

Comment:
These should all be in the `BeforeEach` function at the top of this file.

Reply:
Good point. Done.